### DISCUSSION TITLE
Screen Sharing Work only One Time

### DISCUSSION THREAD

**Discussion by ZinK0** [mylinuxforwork/dotfiles#540] [Q&A]
I'm using zoom meeting and changed the screen sharing for using pipewire but It only work for the first time. After stopping and try to share again it only share black screen and other can't see the screen. I tried with the session restore option but not working. Is there any way to fix that? I new to hyprland and willing to learn that.

**Comment by mylinuxforwork** [comment]
I had the same issue beginning of the week. You need to update all Hyprland related packages to the latest version.

**Reply by ZinK0** [reply]
I update the system everyday. There is no package left behind. I tried with diff kernel and it also happen the same.
